site/docusaurus.config.ts

```diff
@@ -3,230 +3,240 @@ import type { Config } from "@docusaurus/types";
 import { themes as prismThemes } from "prism-react-renderer";
 
 const config: Config = {
-    title: "Ouroboros Leios",
-    tagline: "A high-throughput protocol for Cardano",
-    favicon: "img/wave-logo.ico",
```

Comment:
Must: clean up assets that are to be replaced

This makes it more explicit what to use and not to use (as it's a rebrand).